openedx/core/lib/tests/test_edx_api_utils.py

```diff
@@ -195,3 +196,10 @@ def test_data_retrieval_failure(self, mock_exception):
 
         self.assertTrue(mock_exception.called)
         self.assertEqual(actual, [])
+
+    @mock.patch('openedx.core.lib.edx_api_utils.EdxRestApiClient.__init__')
```

Comment:
@ahsan-ul-haq you've moved this mock, but haven't switched to using a dummy `EdxRestApiClient` in this test.